web/packages/teleterm/src/ui/DocumentCluster/ActionButtons.story.tsx

```diff
@@ -47,9 +49,11 @@ export function ActionButtons() {
 
   return (
     <MockAppContextProvider appContext={appContext}>
-      <VnetContextProvider>
-        <Buttons />
-      </VnetContextProvider>
+      <ConnectionsContextProvider>
```

Comment:
I think I had a fix for that somewhere in the VNet PRs, but they're stuck at the moment. ;f